### DISCUSSION TITLE
NightScout on Google Cloud - update to DEV (15.0.0) version not possible

### DISCUSSION THREAD

**Discussion by streethawk22** [NightscoutFoundation/xDrip#2618] [Q&A]
Hi friends, i have a problem with update the NS from 14.2.6 master version to DEV version 15.0.0.
The Console Windows after 10min wrote a connection lost and the NS site goes not. When I retry the 2nd. step of NS instalation, it goes again, but it is again only the master version.

Who is the problem please? Have it tried from home and now at work PC too and without success.

Error message after update when I try to connect to my site is" 502 Bad gateway".

Thx.

**Comment by Navid200** [comment]
We have not updated the platform to allow updating to the dev branch.

First, please update your platform:  https://navid200.github.io/xDrip/docs/Nightscout/NS_SyncExecutables.html

Then, please run installation phase 1.
Does your system work now?

Please explain why you need to use the development branch.

**Comment by Navid200** [comment]
The capability will be added.  We just need to prioritize it accordingly.  That's why I asked if you could please explain why it is needed now.  I mean there are even people who are still using 14.2.5.
We are already working on testing and enabling 15.0.0 on the platform.

**Comment by streethawk22** [comment]
Hi,

THX for your answer Navid... :)
I use NS for AAPS App and the newest dev version of AAPS has me lastly displaying the message, that I have old version of NS. I don´t know why, but it can be only a bug in AAPS, i wanted to try it with dev version of NS.
I have the new platform, is nothing to up to date. NS version 14.2.6 (master) is working well.

THX...Have a nice day.

**Comment by tzachi-dar** [comment]
@streethawk22 
for making dev work for you, you need to update node to version 16
```
curl -fsSL https://deb.nodesource.com/setup_16.x | sudo -E bash - &&\
sudo apt-get install -y nodejs
```
As @Navid200 has explained, this is still not supported.

**Comment by marionbarker** [comment]
My reason for wanting to use dev on Google Cloud is I'm trying to test a fix to the Day-to-Day plots displaying incorrectly when my time zone does not match that of the Looper's time zone.

I will patch the suggested changes into master instead and test that way.

**Reply by Navid200** [reply]
You can install the development version of Nightscout on Google Cloud now as of yesterday.

**Comment by Navid200** [comment]
In order to install the development branch of Nightscout on Google Cloud you first need to update your platform to the latest.  You can follow this to update your platform to the latest:  https://navid200.github.io/xDrip/docs/Nightscout/NS_SyncExecutables

Then, you can use the customize Nightscout option.  The options is explained here:  https://navid200.github.io/xDrip/docs/Nightscout/Customize.html

The parameters you need to use are the following:
User: nightscout
Repo: cgm-remote-monitor
Branch: dev

**Reply by marionbarker** [reply]
Thank you so much. I’m in the process of building the patched master right now but will try your instructions for dev very soon.

**Reply by streethawk22** [reply]
> In order to install the development branch of Nightscout on Google Cloud you first need to update your platform to the latest. You can follow this to update your platform to the latest: https://navid200.github.io/xDrip/docs/Nightscout/NS_SyncExecutables
> 
> Then, you can use the customize Nightscout option. The options is explained here: https://navid200.github.io/xDrip/docs/Nightscout/Customize.html
> 
> The parameters you need to use are the following: User: nightscout Repo: cgm-remote-monitor Branch: dev

Thank You for your work, I test it today... 👍🏻😉

**Comment by Navid200** [comment]
https://navid200.github.io/xDrip/docs/Nightscout/NightscoutDevBranch.html
